packages/components/src/ConfirmDialog/ConfirmDialog.component.js

```diff
@@ -63,7 +63,7 @@ function ConfirmDialog({
 		if (cancelAction && cancelAction.onClick) {
 			cancelAction.onClick();
 		}
-		return onHide(event);
+		return onHide ? onHide(event) : null;
```

Comment:
you don't need to return anything in this function

Reply:
Linter is unhappy when there is inconsistency with the return statement of a method :/

Reply:
in that case, remove the return?

Reply:
It was there before, didn't want to potentially introduce a breaking change